spec/operators/multicast-spec.ts

```diff
@@ -51,7 +51,9 @@ describe('Observable.prototype.multicast', () => {
 
   it('should accept selectors to factory functions', () => {
     const source =      hot('-1-2-3----4-|');
-    const sourceSubs =     ['^           !'];
+    const sourceSubs =     ['^           !',
+                            '    ^       !',
+                            '        ^   !'];
     const multicasted = source.multicast(() => new Subject(),
       x => x.zip(x, (a, b) => (parseInt(a) + parseInt(b)).toString()));

```

Comment:
@blesh the point of multicast with a selector is to allow multiple subscriptions to the multicast Observable inside the selector function, and one source connection per outer subscriber. By zipping `x` with itself, this test is guaranteeing two subscriptions to `x` for each connection to the multicast source.

I think this is a good test, but I'll add a test with a cold source as well to cover all cases.